docs/how-to/develop-customise/build-kernel-snap.md

```diff
@@ -251,4 +251,4 @@ You are now ready to build the kernel snap.
 
 % LINKS
 [Ubuntu Wiki - Build your own kernel]: https://wiki.ubuntu.com/Kernel/BuildYourOwnKernel
-[Snap - Build environment options]: https://snapcraft.io/docs/build-options#heading--snapcraft
+[Snap - Build environment options]: https://snapcraft.io/docs/build-options
```

Comment:
Thanks @gagath .

This should point to this link though:
https://snapcraft.io/docs/build-options#p-58836-destructive-mode

Though I know there's a risk it might change but the reference here is to be clear about build env + snap base architecture.